readme.md

```diff
@@ -342,6 +342,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Why do C programmers write their letters in *bold*?
```

Comment:
I wonder if this joke would be better to start with something like:

> Why do C programmers break keyboards so often?

Reply:
+1 
Jjajaja, made me laugh just to imagine them typing on those white tall old keyboards

Reply:
Love this, but yeah, it's too similar to the one we already have.